themes/V3/5ePHB/style.less

```diff
@@ -780,16 +780,16 @@ h5 + table{
 	columns    : 1;
 	text-align : center;
 	&:after {
-		all	: unset;
+		display	: none;
 	}
 	h1 {
-		font-family		 : NodestoCapsCondensed;
-		font-weight		 : normal;
-		font-size		   : 2.1cm;
-		margin-top     : 1.2cm;
+		font-family	   : NodestoCapsCondensed;
+		font-weight	   : normal;
+		font-size	   : 2.1cm;
+		margin-top	   : 1.2cm;
```

Comment:
All of the spacing issues should be fixable automatically if you run stylelint on this page. That will probably fix a bunch of other small things too.